acb_dirichlet/zeta_zero.c

```diff
@@ -0,0 +1,748 @@
+/*
+    Copyright (C) 2019 D.H.J. Polymath
```

Comment:
You are very welcome to share your real name (either here or on the credits page in the documentation) if you prefer. Well, it's up to you to be pseudonymous :)

Maybe also include

    Copyright (C) 2010 Juan Arias de Reyna